core/components/atoms/selectionCard/SelectionCard.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { BaseHtmlProps, BaseProps } from '@/utils/types';
+import { useMultiSelect, useSingleSelect } from './hooks';
+
+type ClickEventType = React.MouseEvent<HTMLDivElement> | React.KeyboardEvent;
+
+export interface SelectionCardProps extends BaseProps, BaseHtmlProps<HTMLDivElement> {
+  /**
+   * Element to be rendered inside card item
+   */
+  children: React.ReactNode;
+  /**
+   * Provide unique name to each card when used in group
+   */
+  name?: string;
```

Comment:
Please use value in stead of the name for passing selection value with type String | Object.